src/js/pages/Profile/components/Offer.jsx

```diff
@@ -1,23 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {Row, Col, Button} from 'reactstrap';
-
+import {truncateTwo} from '../../../utils/numbers';
 import {TokenImages} from '../../../utils/images';
 
-const Offer = ({offer, onClick}) => (
-  <Row className="border py-2 mx-0 my-2 rounded">
-    <Col xs="6" className="v-align-center">
+
+const Offer = ({offer, prices, onClick}) => {
+  const price = prices[offer.token.symbol][offer.currency];
+  const calcPrice = offer.marketType === "0" ? price * ((100 + (parseFloat(offer.margin))) / 100) : price * parseFloat(offer.margin) / 100;
```

Comment:
I feel like this formula is gonna get used a few times to calculate the price with the margins, We might want to move it to a utils.